### DISCUSSION TITLE
use <article> tag for articles by default

### DISCUSSION THREAD

**Discussion by yuxi-liu-wired** [quarto-dev/quarto-cli#8048] [Feature Requests]
### Description

Currently `<article>` tag is not used for html documents, but it should be.

**Comment by cderv** [comment]
I believe you are referring to this HTML tag: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article

> The `<article>` [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents a self-contained composition in a document, page, application, or site, which is intended to be independently distributable or reusable (e.g., in syndication). Examples include: a forum post, a magazine or newspaper article, or a blog entry, a product card, a user-submitted comment, an interactive widget or gadget, or any other independent item of content.

For which content are you thinking it should be used in Quarto ? Is this for blog format ? 

Can you give more details and context ? 

Thank you

**Reply by yuxi-liu-wired** [reply]
Thank you for the reply.

Yes, it is for a blog, or generally anything that should be an essay.

For example, in my essay <https://yuxi-liu-wired.github.io/blog/posts/linux-notes/>, I would prefer to wrap the `<article>` tag around `<main class="content quarto-banner-title-block" id="quarto-document-content">`, or perhaps some other method to semantically mark it.

The problem with the current state of affairs is that this hurts machine-readability or accessibility. As an example, currently if I open the page in reader view (I use Postlight Reader, but I suspect this happens for other Readers too), the reader mistakenly grabs the source code of the page and displays it:

![image](https://github.com/quarto-dev/quarto-cli/assets/33951560/c2b406f1-a558-4d81-ac46-03ce167e88bd)

I note that `Distil`-styled articles have the `<d-article>` tag, but the benefit of inventing another tag is unclear, when `<article>` would serve just as well.

**Reply by cderv** [reply]
Thanks for the additional information ! 

@dragonstyle did we consider using `<article>` for blog ? If so, is there a reason not too ? 

Otherwise, this is probably a good feature request to consider or good use case to push the custom template customisation for blog article maybe ... 🤔

**Reply by dragonstyle** [reply]
We didn't consider it, though I think this is a nice feature request. We'd likely want a YAML option of some kind (since many forms of HTML content produced with Quarto might not warrant this treatment) and would want to default it on for blogs...

**Reply by mcanouil** [reply]
Follow up on:
- https://github.com/quarto-dev/quarto-cli/issues/8100
